conf/messages

```diff
@@ -5,5 +5,12 @@ index.use_the_quick_calc_if=Use the quick calculation if you can answer ’yes
 index.employment_only=you have employment income only
 index.dont_contribute_pension=you don’t contribute to a workplace or personal pension through your employer.
 
+quick_calc.title=Do you know your Tax Code?
```

Comment:
sorry to be picky, this message belongs to about_tax_code and should be namespaced as such